misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/IterableContainsInOrderOnlyGroupedEntriesAssertionsSpec.kt

```diff
@@ -267,17 +264,17 @@ abstract class IterableContainsInOrderOnlyGroupedEntriesAssertionsSpec(
                                 contains.exactly(1).value("$rootBulletPoint$containsInOrderOnlyGrouped:")
                                 indexFail(
                                     0, 1, listOf(1.0, 2.0),
+                                    sizeCheck(2, 2),
```

Comment:
note, if the size is correct, then it will no longer be shown the user as it is no longer part of the AssertionGroup with a SummaryAssertionGroupType (which has the effect that it shows also assertions which hold).
In short, here you should actually check that the size check does no longer show up. See ch.tutteli.atrium.specs.integration.MapContainsInAnyOrderOnlyKeyValuePairsAssertionsSpec => Line 110 as an example

Reply:
Managed to do that but in Grouped assertions it still prints out it out, in short i get something like this:
`◆ contains only, in order, grouped: 
  ✘ ▶ index 0..1: [null, 1.0]        (java.util.ArrayList.SubList <1355742676>)
      ◾ ▶ size: 2        (kotlin.Int <1639154810>)
          ◾ equals: 2        (kotlin.Int <1639154810>)
      ◾ contains only, in any order: 
        ✔ an element which: 
            » is: null
        ✘ an element which: 
            » is: null
        ❗❗ following elements were mismatched: 
           ⚬ 1.0        (kotlin.Double <459559002>)
  ✘ ▶ index 2..3: [null, 3.0]        (java.util.ArrayList.SubList <1533221659>)
      ◾ ▶ size: 2        (kotlin.Int <1639154810>)
          ◾ equals: 2        (kotlin.Int <1639154810>)
      ◾ contains only, in any order: 
        ✔ an element which: 
            » is less than: 5.0        (kotlin.Double <1702487214>)
        ✘ an element which: 
            » is greater than: 2.0        (kotlin.Double <1123976584>)
        ❗❗ following elements were mismatched: 
           ⚬ null)`

Reply:
That's fine for the moment. The grouped assertions use itself SummaryAssertionGroupType (thus the ✘ in front of index...) and as they are implemented currently, they show all sub-assertions in case they fail (see also https://github.com/robstoll/atrium/issues/724)